src/Database.php

```diff
@@ -136,6 +136,19 @@ public function dropCollection($collectionName)
     }
 
     /**
+     * This is the method to issue database commands.
+     *
+     * @param Command $command A database command to send
+     * @param ReadPreference|null $readPreference Default read preference to apply
+     * @return Cursor
+     */
+    public function executeCommand(Command $command, ReadPreference $readPreference = null)
+    {
+        $server = $this->manager->selectServer(new ReadPreference(ReadPreference::RP_PRIMARY));
+        return $server->executeCommand($this->databaseName, $command, $readPreference);
```

Comment:
I'll discuss this with @derickr this week, but I think we may want to change two things here:

 * The command document can be an array or object (i.e. a document), which we'll use to construct a Command instance internally. In the library, I don't think we need to require the driver value object to be used (for the convenience of users).
 * This appears to be hard-coded to select the primary server. I think we'll want to take an optional read preference and then use that to select the server, as we do in [`Collection::count()`](https://github.com/mongodb/mongo-php-library/blob/6faa366281f68f5e4f24af73dec81d3cd55217db/src/Collection.php#L167).

Feel free to hold off on any chances until I follow up, though. I don't want to waste your time on this.